tests/leaking-vars/compare.js

```diff
@@ -0,0 +1,16 @@
+var expect = require('expect.js');
+var fs = require('fs');
+var util = require('util');
+
+var sample1 = fs.readFileSync('tests/leaking-vars/output1.html').toString();
+var sample2 = fs.readFileSync('tests/leaking-vars/output1.html').toString();
+var test1 = fs.readFileSync('tests/leaking-vars/_output2.html').toString();
+var test2 = fs.readFileSync('tests/leaking-vars/_output2.html').toString();
```

Comment:
we need to compare `output1` width `_output1` and `output2` with `_output2`.

```
var sample1 = fs.readFileSync('tests/leaking-vars/output1.html').toString();
var sample2 = fs.readFileSync('tests/leaking-vars/output2.html').toString();
var test1 = fs.readFileSync('tests/leaking-vars/_output1.html').toString();
var test2 = fs.readFileSync('tests/leaking-vars/_output2.html').toString();
```

?